catalog/USFS/USFS_GTAC_LCMS_v2022-8.jsonnet

```diff
@@ -0,0 +1,824 @@
+local id = 'USFS/GTAC/LCMS/v2022-8';
+local latest_id = id;
+local predecessor_id = 'USFS/GTAC/LCMS/v2021-7';
+local subdir = 'USFS';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+
+local license = spdx.proprietary;
+
+local basename = std.strReplace(id, '/', '_');
+local latest_basename = std.strReplace(latest_id, '/', '_');
+local predecessor_basename = std.strReplace(predecessor_id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+local catalog_subdir_url = ee_const.catalog_base + subdir + '/';
+
+{
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+    ee_const.ext_ver,
+  ],
+  id: id,
+  title: 'USFS Landscape Change Monitoring System v2022.8 (Conterminous United States and Southeastern Alaska)',
+  version: 'v2022.8',
```

Comment:
Unless `v` is a part of the actual version number, please remove it. And if it's the case the the v is not a part of the version, I'll remove it from the old ones. Typically the `v` is only there to make it obvious that those characters following are a version, e.g. in a title or asset id